buf.yaml

```diff
@@ -0,0 +1,19 @@
+version: v2
+modules:
+  - path: internal/proto
+lint:
+  use:
+    - DEFAULT
+  except:
+    - FIELD_NOT_REQUIRED
+    - PACKAGE_NO_IMPORT_CYCLE
```

Comment:
We didn't need these before. Why do we need them now?

Reply:
We don't - they're excluded as part of `buf config migrate` as they aren't part of the `v1` `DEFAULT` lint rules, but I've removed them and verified we're passing them. We could make `buf config migrate` smarter to test a build without them, but that was too much.

Reply:
We could have the migrate command include a comment for each of these that says something to the effect of "This is a new lint rule this is on by default, but is disabled to ensure a backward compatible migration. Recommend removing these exceptions and fixing your build if it breaks"

Reply:
Ah, I'd forgotten about that discussion. Mea culpa.

Including a comment in the migration output would be nice.

Reply:
Comments are...difficult in yaml I think, given our setup, but we could look into it